src/zenml/model/model_config.py

```diff
@@ -30,24 +38,182 @@
 logger = get_logger(__name__)
 
 
-class ModelConfig(ModelConfigModel):
-    """ModelConfig class to pass into pipeline or step to set it into a model context.
+class ModelConfig(BaseModel):
+    """ModelConfig class to pass into pipeline or step to set it into a model context."""
 
-    name: The name of the model.
-    version: The model version name, number or stage is optional and points model context
-        to a specific version/stage, if skipped and `create_new_model_version` is False -
-        latest model version will be used.
-    version_description: The description of the model version.
-    create_new_model_version: Whether to create a new model version during execution
-    save_models_to_registry: Whether to save all ModelArtifacts to Model Registry,
-        if available in active stack.
-    delete_new_version_on_failure: Whether to delete failed runs with new versions for later recovery from it.
-    """
+    name: str
+    license: Optional[str]
+    description: Optional[str]
+    audience: Optional[str]
+    use_cases: Optional[str]
+    limitations: Optional[str]
+    trade_offs: Optional[str]
+    ethic: Optional[str]
```

Comment:
```suggestion
    ethics: Optional[str]
```

Probably should be this (and below in relevant places)

Reply:
Done